packages/tenderly-actions/src/actions/processRiskParameterUpdate.ts

```diff
@@ -0,0 +1,49 @@
+import { ActionFn, Context, Event, TransactionEvent } from "@tenderly/actions";
+
+import { getTelegramClient } from "./clients/telegram";
+import { getRiskOracleInterface, getRiskStewardReceiverContract } from "./contracts";
+import messages from "./messages";
+import { getNetworkConfig } from "./networkConfig";
+import { getGateways } from "./utils/gateway";
+
+export const getRiskParameterUpdateEvent = (
+  event: Event,
+): {
+  updateId: string;
+} => {
+  const txEvent = event as TransactionEvent;
+  const iface = getRiskOracleInterface();
+
```

Comment:
we can use log validation 
`
if (txEvent.logs.length === 0) {
  throw new Error("No logs in transaction");
}
`